client/components/external-link/README.md

```diff
@@ -27,3 +27,43 @@ The following props can be passed to the External Link component:
 ## Other Props
 Any other props that you pass into the `a` tag will be rendered as expected.
 For example `onClick` and `href`.
+
+External Link with Tracking
+=======
+
+External Link with Tracking is a React component for rendering an external link that is connected to the Redux store
+and is capable of recording Tracks events.
+
+## Usage
+
+```jsx
+
+import React from 'react';
+import ExternalLinkWithTracking from 'components/external-link/with-tracking';
+
+class MyComponent extends React.Component {
+	render() {
+		return (
+			<ExternalLinkWithTracking
+				icon={ true }
+				href="https://wordpress.org"
+				onClick="somefunction()"
```

Comment:
This doesn't seem right - should this be an arrow function?

Reply:
Actually I just copy-pasted from another example above. I'll fix it in both locations.